bitrise.yml

```diff
@@ -200,14 +208,22 @@ workflows:
             #!/bin/bash
             set -ex
             git clone --depth 1 $SAMPLE_ARTIFACTS_GIT_CLONE_URL ./_tmp
-  test:
-    before_run:
-    - download_sample_artifacts
+
+  _test_results_deployment:
```

Comment:
This is more like a smoke test, that helped verify the log changes, see `(1) Test result deployment` in the CI build log.